event_registration_deliverable_product/README.rst

```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Product Delivery for Event Registrations
+========================================
+
+Sometimes you want to give some products to your event's attendees. For
+example: a pen, a notebook, a book, etc.
+
+This module extends the functionality of events to support setting products to
+deliver to event attendees and allow you to know if they have been already
+delivered to them.
+
+.. warning::
+    Please do not get confused between *event products* and *products to
+    deliver in events*:
+
+    * **Event products** are created in the ``event_product`` addon, and
+      define an event as a product, to have a catalog of them.
+    * **Products to deliver in events** are created in this addon, and define
+      the products that you will have to deliver to event attendees.
+
+      They can be defined in the *event product*, in the event directly, or in
+      both.
```

Comment:
Is it me or this reads the same as in the `event_product_logistics`? 
I then get the idea that both modules do the same.